CONTRIBUTING.md

```diff
@@ -188,6 +189,10 @@ You can then find the generated HTML files in `build/documentation/html`.
 
 Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels ((enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/aws/amazon-braket-sdk-python/labels/help%20wanted) issues is a great place to start.
 
+## Building Integrations
+Amazon Braket SDK supports integrations with popular quantum computing frameworks such as [PennyLane](https://github.com/aws/amazon-braket-pennylane-plugin-python), [Strawberryfields](https://github.com/aws/amazon-braket-strawberryfields-plugin-python) and [DWave's Ocean library](https://github.com/aws/amazon-braket-ocean-plugin-python). These serve as a good reference for a new integration you wish to develop. 
```

Comment:
nit: I think we refer to it as The Amazon Braket SDK not Amazon Braket SDK other places.

Reply:
Yes, thanks. Updated.